### DISCUSSION TITLE
How to redirect to new URL when use getServerSideProps.

### DISCUSSION THREAD

**Discussion by hieusmiths** [vercel/next.js#14436] [Ideas]
Hm. It's my ide haha.
Okay let'go now i'm using getAnyFuncNextJS({ ..., request, response)) {
  Hmm.
  Return res.redirect(/)
}:

This is idea has possiable ?

Thank so much team.

**Comment by kennethacohen** [comment]
Not sure I fully understand, but check out this thread for some patterns on redirecting from getServerSideProps:

https://github.com/vercel/next.js/discussions/11281
